src/tcgwars/logic/impl/gen3/Deoxys.groovy

```diff
@@ -1283,12 +1291,7 @@ public enum Deoxys implements LogicCardInfo {
             text "Look at the top 5 cards of either player’s deck and put them back on top of that player’s deck in any order."
             energyCost C
             attackRequirement {}
```

Comment:
Probably a good idea to remove this too, as `rearrangeEitherPlayersDeck` explicitly defines one.

Reply:
Same, should be corrected in the static.

Reply:
What's to be corrected in the static? This is purely so someone doesn't attempt to add the attackRequirements here before reading what this method actually does, since it already handles it.

Reply:
Ah, based off your latest commit, I see what you meant, but I think you misunderstood what I meant. I chose the line this comment is on very specifically :laughing:

Reply:
Ahh, ok yeah I get what you mean. Will fix it in a bit!